cloudvolume/cloudvolume.py

```diff
@@ -284,11 +287,26 @@ def create_new_info(cls,
         "size": list(map(int, volume_size)),
       }],
     }
-   
-    # add mip levels
+    
+    def downscale(size, factor_in_mip, roundingfn):
```

Comment:
I don't like replicating this logic by copy paste. Would it make more sense to refactor this into a function and use it here and in add_scale?

Reply:
I don't either. I should follow the principles of clean code. 
moved this function outside the class.